src/components/create/MultipleChoices.tsx

```diff
@@ -36,6 +41,10 @@ const MultipleChoices = ({
 }: Props): JSX.Element => {
   const { t } = useTranslation();
 
+  const [explanationList, setExplanationList] = useState<List<boolean>>(
+    List(choices.map((choice) => (choice.explanation ? true : false)))
```

Comment:
```suggestion
    List(choices.map((choice) => Boolean(choice.explanation)))
```

Reply:
I don't get why you need this list?? Why not just check on `choice.explanation`?

Reply:
Hi Kim! Further down the implementation line I had the same insight, but I realized I needed it to display the explanation text field when the add answer button is pressed (and choice.explanation is empty).